### DISCUSSION TITLE
Highlight the "reserved" labels (HTML ids) prefixes

### DISCUSSION THREAD

**Discussion by mcanouil** [quarto-dev/quarto-cli#3044] [Feature Requests]
Recently and few times already, some issues were reported when using some of the prefixes defined in Quarto for cross-references.
- <https://twitter.com/PipingHotData/status/1585067723268050944?s=20&t=QMv2ClFJdoD8W6cWpy0-5A>
- <https://twitter.com/LisaDeBruine/status/1584586868058132481?s=20&t=QMv2ClFJdoD8W6cWpy0-5A>

I think, it would be a good idea to highlight somewhere (maybe in <https://quarto.org/docs/authoring/cross-references.html>) that there are some "reserved" prefixes for labels and HTML ids, such as `fig-`, `tbl-`, `thm-`, `eq-`, _etc._

**Comment by cscheid** [comment]
For sure. We'll be doing some major work on those starting on 1.3. One thing we could do is emit a warning if the content of figures/etc doesn't match our expectations. Right now we're silently failing most of the time.

**Comment by mcanouil** [comment]
Closing in favour of <https://github.com/quarto-dev/quarto-cli/issues/6201>.
